receiver/statsdreceiver/transport/client/client.go

```diff
@@ -83,11 +64,8 @@ func (s *StatsD) Disconnect() error {
 
 // SendMetric sends the input metric to the StatsD connection.
 func (s *StatsD) SendMetric(metric Metric) error {
-	_, err := fmt.Fprint(s.Conn, metric.String())
-	if err != nil {
-		return err
-	}
-	return nil
+	_, err := io.Copy(s.Conn, strings.NewReader(metric.String()))
+	return err
```

Comment:
wrapping the error?

If it used merely in tests I would add a comment specifying it

Reply:
👍